.changeset/spicy-donuts-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/text-input': patch
+---
+
+Fixes a regression. Now when `type="search"`, providing `aria-label` is sufficient for accessibility
```

Comment:
Aria-label should always be sufficient for accessibility as it relates to TextInput components